src/components/Signup/Signup.js

```diff
@@ -1,5 +1,98 @@
 import React from "react";
-
+import { Navbar, Footer } from "../components";
+import { Link } from "react-router-dom";
 export default function Signup() {
-  return <div>Signup</div>;
+  return (
+    <div>
+      <Navbar />
```

Comment:
You can extract out the navbar in the the app.js file. As navbar will be there in all the pages. So no need to render it with the pages.